third_party/xla/xla/backends/profiler/gpu/rocm_collector.cc

```diff
@@ -223,7 +223,7 @@ class PerDeviceCollector {
     stats.occupancy_pct /= device_properties_.maxThreadsPerMultiProcessor;
 
     err = hipOccupancyMaxPotentialBlockSize(
```

Comment:
could you also upstream this change as well?

Reply:
@zahiqbal mentioned he is already doing some changes in profiler, and that he'll add this to his PR as well

Reply:
Probably they don't get an error here due to jax build using gcc. Guess we need to check those changes applied to tensorflow-upstream to be sure.

Reply:
Yes, I realise is clang caused this error on TF this morning as well..